docs/webserver/overview.md

```diff
@@ -0,0 +1,17 @@
+# iRobot® Create® 3 Webserver Overview
+The Create® 3 robot runs a configuration webserver for modifying settings of the robot.
+This is a separate process from its ROS 2[^1] application; it is running regardless of the status of that application.
```

Comment:
I would make it explicit that the webserver can be accessed even if the ros app is not running